lib/command/src/Obelisk/Command/Project.hs

```diff
@@ -66,6 +65,11 @@ import System.PosixCompat.User
 import qualified System.Process as Proc
 import Text.ShellEscape (sh, bash, bytes)
 
+#if !MIN_VERSION_base(4,18,0)
```

Comment:
Any reason we're using `base` here but used `__GLASGOW_HASKELL__ >= 906` above?
Maybe it'd be clearer if we used `MIN_VERSION_mtl(2,3,0)`

Reply:
Because the root of the error/warning is the change in the `base` library. Although ghc versions and base go hand-in-hand, I preferred addressing the root cause.